src/common/Banner.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2016+     AzerothCore <www.azerothcore.org>, released under GNU GPL v2 license, you may redistribute it and/or modify it under version 2 of the License, or (at your option), any later version.
+ * Copyright (C) 2008-2016 TrinityCore <http://www.trinitycore.org/>
```

Comment:
I think we can remove TC from here

Reply:
only reason I put it there is because this is basically a copy-paste from TC but with our banner

Reply:
thoguhts?

Reply:
^ Me trying to figure out what that means O.o

Reply:
for me it's the same, we can keep it or remove
this project is clearly based on TC, so it does not matter I think